### ISSUE TITLE
[question] can i modify https and http ports

### ISSUE THREAD

**Issue by tuxsu** [ZoeyVid/NPMplus#1318]
my ISP has disabled ports 80 and 443, and I'm using docker deployment (``network_mode: host``); can I change ports 80 and 443 to other ports?

**Comment by Zoey2936** [comment]
you can map 443 and 80 to other ports in your router, but then acme http challenges wont work and http3 will not work, but I'm currently working on a way to make http3 work with other ports

**Comment by tuxsu** [comment]
Thanks

But I found that the ``/api/reports/hosts``  api will be requested all the time on the home page;

**Comment by Zoey2936** [comment]
yes this error was already fixed in develop (currently broken) and also exists in upstream: https://github.com/NginxProxyManager/nginx-proxy-manager/pull/4246

**Comment by Zoey2936** [comment]
I will close this since it is done and will be included in the next release

**Comment by Zoey2936** [comment]
if you want, you can test this alpha/prerelease: https://github.com/ZoeyVid/NPMplus/releases/tag/2025-01-03-alpha (but make a backup first)

**Comment by tuxsu** [comment]
> if you want, you can test this alpha/prerelease: https://github.com/ZoeyVid/NPMplus/releases/tag/2025-01-03-alpha (but make a backup first)

thank you
